web/src/main/java/org/reciplease/configuration/LocalSecurityConfig.java

```diff
@@ -1,21 +1,41 @@
 package org.reciplease.configuration;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.CorsConfigurationSource;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.Arrays;
 
 @Profile("local")
 @Configuration
 @EnableWebSecurity
-public class LocalSecurityConfig extends WebSecurityConfigurerAdapter {
+public class LocalSecurityConfig extends WebSecurityConfigurerAdapter implements WebMvcConfigurer {
+
+	@Value("${RECIPLEASE_WEB_API_URL}")
+	private String webApiURL;
 
     @Override
     protected void configure(final HttpSecurity security) throws Exception {
         security.httpBasic().disable()
-                .csrf().disable();
+                .csrf().disable().cors();
 
         security.headers().frameOptions().disable();
     }
+
+    @Bean
+    CorsConfigurationSource corsConfigurationSource() {
+        CorsConfiguration configuration = new CorsConfiguration();
+        configuration.setAllowedOrigins(Arrays.asList(webApiURL));
```

Comment:
`Arrays.asList`
prefer `List.of`

Reply:
This is throwing an error on my SonarQube Linting. Should I ignore it?
```
Classes from "sun.*" packages should not be used
 
Code smell
 
Major
squid:S1191
 
Classes in the sun.* or com.sun.* packages are considered implementation details, and are not part of the Java API.
They can cause problems when moving to new versions of Java because there is no backwards compatibility guarantee. Similarly, they can cause problems when moving to a different Java vendor, such as OpenJDK.
Such classes are almost always wrapped by Java API classes that should be used instead.
```

Reply:
might be `Lists.of`